README.md

```diff
@@ -1,94 +1,93 @@
 [![Build Status](https://travis-ci.org/wso2-ballerina/module-mongodb.svg?branch=master)](https://travis-ci.org/wso2-ballerina/module-mongodb)
 
-# Ballerina MongoDB Client
+Ballerina MongoDB Client is used to connect Ballerina with MongoDB data source.<br/> <br/>
+ With the Ballerina MongoDB client following operations are supported.
 
-Ballerina MongoDB Client is used to connect Ballerina with MongoDB data source. With the Ballerina MongoDB client following operations are supported.
-
-1. insert - To insert document to a given collection
-2. find - To select document from a given collection according to given query.
-3. findOne - To select the first document match with the query.
-4. update - To update documents that matches to the given filter.
+1. insert - To insert document to a given collection <br/>
+2. find - To select document from a given collection according to given query. <br/>
+3. findOne - To select the first document match with the query.<br/>
+4. update - To update documents that matches to the given filter. <br/>
 5. delete - To delete documents that matches to the given filter.
+Steps to Configure <br/>
 
-Steps to Configure
-==================================
-
-Extract wso2-mongodb-<version>.zip and  Run the install.sh script to install the module.
-You can uninstall the module by running uninstall.sh.
-
-Building From the Source
-==================================
-If you want to build Ballerina MongoDB client from the source code:
+## Samples
 
-1. Get a clone or download the source from this repository:
-    https://github.com/wso2-ballerina/module-mongodb
-2. Run the following Maven command from the ballerina directory: 
-    mvn clean install
-3. Extract the distribution created at `/component/target/wso2-mongodb-<version>.zip`. Run the install.{sh/bat} script to install the module.
-You can uninstall the module by running uninstall.{sh/bat}.
+### Performing CRUD operations with MongoDB client
 
-Sample
-==================================
+Following is a simple Ballerina program that can be used to perform CRUD operations.
 
 ```ballerina
-import wso2/mongodb;
 import ballerina/io;
```

Comment:
You have to delete the `ballerina/io` import since it is unused after the below modifications.

Reply:
Since io:sprintf is used in the sample, let's keep the import statement.